packages/manager/apps/pci-public-ip/src/pages/order/hooks/data/useOrderCatalog.ts

```diff
@@ -0,0 +1,92 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { useQuery } from '@tanstack/react-query';
+import { useMe } from '@/pages/order/hooks/data/useMe';
+
+export const getOrderCatalog = async ({
+  genre,
+  type,
+  ovhSubsidiary,
+  productName,
+}: {
+  genre: 'public' | 'formatted';
+  type: 'cloud' | 'ip';
+  ovhSubsidiary: string;
+  productName: string;
+}): Promise<any> => {
+  const { data } = await v6(
```

Comment:
```suggestion
  const { data } = await v6.get(
```

Reply:
it worked like that, i believe that in this case V6 default to get, gonna change that